### PR TITLE
Add OpenFaaS Clear Linux based GO template

### PR DESCRIPTION
**Pull request by sophia0328**
Signed-off-by: Sophia Gong <sophia.gong@intel.com>

### CODE REVIEW

**Comment by sophia0328** [comment]
@ZhongbaoShi , @qzheng527 , please take a review for GO faas template patch.

File: FaaS/OpenFaaS/template/go-clearlinux/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
+FROM openfaas/classic-watchdog:0.18.1 as watchdog
+
+#FROM golang:1.11-alpine3.10 as builder
+FROM clearlinux/golang as builder
+
+# Allows you to add additional packages via build-arg
+ARG ADDITIONAL_PACKAGE
```
Comment (qzheng527) [inline FaaS/OpenFaaS/template/go-clearlinux/Dockerfile]: @sophia0328 Clear Linux doesn't use this arg, any reason to set it here?

Comment (sophia0328) [reply FaaS/OpenFaaS/template/go-clearlinux/Dockerfile]: check with official GO faas template, it seems this is not necessary. will remove this line.

**Comment by sophia0328** [comment]
please review with https://github.com/clearlinux/dockerfiles/pull/328

**Pull request closed without merging**